howto/setup-secret-store/json-secret-store.md

```diff
@@ -0,0 +1,104 @@
+# Secret Store for JSON (Development)
+
+This document shows how to enable JSON secret store using [Dapr Secrets Component](../../concepts/secrets/README.md) for Development scenarios in Standalone mode. This Dapr secret store component reads plain text JSON from a given file and does not use authentication.
+
+## Contents
+
+- [Create JSON file to hold the secrets](#create-json-file-to-hold-the-secrets)
+- [Use JSON secret store in Standalone mode](#use-json-secret-store-in-standalone-mode)
+- [References](#references)
+
+## Create JSON file to hold the secrets
+
+This creates new JSON file to hold the secrets.
+
+1. Create a json file (i.e. secrets.json) with the following contents:
+
+```json
+{
+    "redisPassword": "your redis passphrase"
+}
+```
+
+## Use JSON secret store in Standalone mode
+
+This section walks you through how to enable an JSON secret store to store a password to access a Redis state store in Standalone mode.
+
+1. Create a components directory in your application root
+
+All Dapr components are stored in a directory called 'components' below at application root. Create this directory.
+
+```bash
+mkdir components
+```
+
+2. Create a file called jsonsecretstore.yaml in the components directory
+
+Now create an Dapr jsonsecretstore component. Create a file called jsonsecretstore.yaml in the components directory with the content below
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: jsonsecretstore
+  namespace: default
+spec:
+  type: secretstores.json.jsonsecretstore
+  metadata:
+  - name: secretsFile
+    value: [path to the secrets.json you created in the previous section]
+  - name: nestedSeparator
+    value: ":"
+```
+
+3. Create redis.yaml in the components directory with the content below
+
+Create a statestore component file. This Redis component yaml shows how to use the `redisPassword` secret stored in a JSON secret store called jsonsecretstore as a Redis connection password.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: statestore
+  namespace: default
+spec:
+  type: state.redis
+  metadata:
+  - name: redisHost
+    value: "[redis]:6379"
+  - name: redisPassword
+    secretKeyRef:
+      name: redisPassword
+      key: redisPassword
+auth:
+    secretStore: jsonsecretstore
+```
+
+4. Run your app
+
+You can check that `secretstores.json.jsonsecretstore` component is loaded and redis server connects successfully by looking at the log output when using the dapr `run` command with the `--enable-json-secretstore` flag.
+
+> The JSON secret store is in no way recommended for production environments and therefore is disabled by default. Using the `--enable-json-secretstore` flag sets the **DAPR_ENABLE_JSON_SECRET_STORE** environmnet variable to **1** so the store is enabled in your developemnt or demo box.
```

Comment:
This is an important disclaimer, please move it or repeat it at the top of the article

Reply:
Moved to the top